core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkspaceComponent.scala

```diff
@@ -326,6 +326,11 @@ trait WorkspaceComponent {
     ): ReadAction[Option[Workspace]] =
       loadWorkspace(findByNameQuery(workspaceName), attributeSpecs)
 
+    def findV2WorkspaceByName(workspaceName: WorkspaceName,
+                              attributeSpecs: Option[WorkspaceAttributeSpecs] = None
+    ): ReadAction[Option[Workspace]] =
+      loadWorkspace(findV2WorkspaceByNameQuery(workspaceName), attributeSpecs)
+
     def findById(workspaceId: String): ReadAction[Option[Workspace]] =
```

Comment:
This is still being called by the `CloneWorkspaceContainerRunner` which should probably switch to `findV2WorkspaceById` although sort of a nonissue since that runner will theoretically only ever touch v2 workspaces

Reply:
Right, I'm not trying to prevent all internal usages to the original APIs… I tried that first and it was a mess. Instead I'm just preventing v1 workspace access/usage at the service layer. In addition to the fact that the runner only works on Azure workspaces, clone operations can't be started on v1 workspaces.